### ISSUE TITLE
Require services for railway infrastructure

### ISSUE THREAD

**Issue by chalwachalwa** [robs74/offices-generate-money#8]
What do you think about adding additional cost of paper work for railway stations, depots, RCO, RDO etc. ? That might be resolved building signaling towers taking buorocrats as workers and railway offices.

**Comment by robs74** [comment]
Yeah, it's a nice idea. It could also be extended to road and water transport. It's not covered by workers in the base game and this would make logistics actually require some workforce.
